playbooks/aws/openshift-master/config.yml

```diff
@@ -35,6 +35,7 @@
     - {
         role: ../../../roles/openshift_master,
         oo_node_ips: "{{ hostvars['localhost'].oo_node_ips | default(['']) }}",
-        oo_bind_ip: "{{ hostvars[inventory_hostname].ansible_eth0.ipv4.address | default(['']) }}"
+        oo_bind_ip: "{{ hostvars[inventory_hostname].ansible_eth0.ipv4.address | default(['']) }}",
+        oo_public_ip: "{{ hostvars[inventory_hostname].facter_ec2_public_ipv4 }}"
```

Comment:
This should be using hostvars[inventory_hostname].ec2_ip_address instead. Otherwise we are introducing a new dependency on facter.